source/includes/steps-install-bi-connector-rhel.yaml

```diff
@@ -6,6 +6,72 @@ source:
   ref: download-bi
 ---
 stepnum: 2
+title: Verify the integrity of the downloaded package.
+ref: verify-bi-rhel-package
+level: 4
+content: |
+  The MongoDB release team digitally signs all software packages to
+  certify that a particular MongoDB package is a valid and unaltered
+  MongoDB release. MongoDB signs each release branch with a different
+  PGP key in ``.asc`` format.  
+
+  a. Run the following command to downloaded the ``.sig`` file.
+       
+     .. code-block:: sh 
+            
+        curl -LO https://info-mongodb-com.s3.amazonaws.com/mongodb-bi/v2/mongodb-bi-linux-{arch}-{platform}-{version}.tgz.sig
```

Comment:
Maybe we could add an example here. I think it won't be super straightforward for users to figure out the arch and platform.
Here is an example link `https://info-mongodb-com.s3.amazonaws.com/mongodb-bi/v2/mongodb-bi-linux-x86_64-rhel90-v2.14.14.tgz.sig`